### DISCUSSION TITLE
Rename 'index' translation files

### DISCUSSION THREAD

**Discussion by siliconeidolon** [ivanhofer/typesafe-i18n#732] [Q&A] [answered]
Hello again. We often use `cmd+p` to navigate between files in Vscode and it'd be awesome to input `en`/`de` and find the translation file, but unfortunately all the translation files must be named `index.ts` in order for the generator to find them. 

Is there somewhere in the config that I can set custom paths for the locale files so I can name them `en.ts/de.ts etc`?

My current workaround is to have a `de.ts/en.ts` file, import the translations object in the `index.ts` file, then export it as default. This works nicely, so no worries if there isn't a simple way to set a custom path. 😆

**Comment by ivanhofer** [comment, marked as answer]
The convention to use `index.ts` makes it possible to co-locate and group translations inside a folder. It is a feature I don't really want to remove if there is no better recommandation on how to achieve that functionality.

When using VS Code you can set `"workbench.editor.labelFormat": "short"` to see a small label with the `locale` name next to the file name
![image](https://github.com/ivanhofer/typesafe-i18n/assets/21335119/4d1dc432-fc55-44c7-8f47-6899f0731fb5)
